### PR TITLE
[docs] Paying attention on multi-master quorum

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Paying attention on multi-master quorum in docs.

## Why do we need it, and what problem does it solve?
Not everyone knows about quorum.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Paying attention on multi-master quorum in docs.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ title: "Managing control plane: FAQ"
...
 ## How do I add a master node to a static or hybrid cluster?
 
+> It is recommended to have an odd number of masters to ensure a quorum.
+
 Adding a master node to a static or hybrid cluster has no difference from adding a regular node to a cluster. To do this, use the corresponding [instruction](../040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster). All the necessary actions to configure a cluster control plane components on the new master nodes are performed automatically. Wait until the master nodes appear in `Ready` status.
 
 ## How do I add a master nodes to a cloud cluster (single-master to a multi-master)?
 
 > Before adding nodes, ensure you have the required quotas in the cloud provider.
+<!-- br https://github.com/DavidAnson/markdownlint/issues/263 -->
```
Comment (apolovov) [inline modules/040-control-plane-manager/docs/FAQ.md:16]: @z9r5 halp!

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 2]
```diff
@@ -6,11 +6,15 @@ title: "Managing control plane: FAQ"
...
+> It is recommended to have an odd number of masters to ensure a quorum.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:17]:
```suggestion
> Before adding nodes, ensure you have the required quotas in the cloud provider. <!-- br https://github.com/DavidAnson/markdownlint/issues/263 -->
> 
> It is recommended to have an odd number of masters to ensure a quorum.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -6,11 +6,15 @@ title: "Управление control plane: FAQ"
...
 
+> Рекомендуется иметь нечётное количество мастеров для обеспечения кворума.
+
 Добавление master-узла в статичный или гибридный кластер ничем не отличается от добавления обычного узла в кластер. Воспользуйтесь для этого соответствующей [инструкцией](../040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер). Все необходимые действия по настройке компонентов control plane кластера на новом узле будут выполнены автоматически, дождитесь их завершения — появления master-узлов в статусе `Ready`.
 
 ## Как добавить master-узлы в облачном кластере (single-master в multi-master)?
 
 > Перед добавлением узлов убедитесь в наличии необходимых квот.
+<!-- br https://github.com/DavidAnson/markdownlint/issues/263 -->
+> Рекомендуется иметь нечётное количество мастеров для обеспечения кворума.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:17]:
```suggestion
> Перед добавлением узлов убедитесь в наличии необходимых квот. <!-- br https://github.com/DavidAnson/markdownlint/issues/263 -->
>
> Рекомендуется иметь нечётное количество мастеров для обеспечения кворума.
```

File: modules/040-control-plane-manager/docs/FAQ.md (hunk 1 above)
Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ.md:16]: Added suggestion

**Review by z9r5 — APPROVED** [review]

File: candi/cloud-providers/aws/openapi/cluster_configuration.yaml [hunk 4]
```diff
@@ -32,7 +32,7 @@ apiVersions:
             type: integer
             minimum: 1
             description: |
-              The number of master nodes to create.
+              The number of master nodes to create. It is recommended to have an odd number of masters to ensure a quorum.
```
Comment (RomanenkoDenys) [inline candi/cloud-providers/aws/openapi/cluster_configuration.yaml:35]: Why recommended ? It is mandatory requirement.

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
needs some attention

File: candi/cloud-providers/aws/openapi/cluster_configuration.yaml (hunk 4 above)
Comment (apolovov) [reply candi/cloud-providers/aws/openapi/cluster_configuration.yaml]: It isn't mandatory cause we don't control this technically. I switched recommended to important.

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit 732b8dba24b3122d90638400e84dd4d4e206e129 to the branch [release-1.42](https://github.com/deckhouse/deckhouse/tree/release-1.42) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/3927515442) for details.

**Comment by z9r5** [comment]
Cherry-picked into release-1.42

**Pull request merged**